<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 114, 255);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Departement</string>
   </property>
  </widget>
  <widget class="QLabel" name="imglbl">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nama </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>NIP</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Daftar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string>Daftar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrasi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonToAbsen">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>370</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string>Halaman Absensi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
